cmd/kubectl-gadget/snapshot/process.go

```diff
@@ -41,14 +41,14 @@ func newProcessCmd() *cobra.Command {
 		},
 	}
 
-	availableColumns := map[string]struct{}{
-		"node":      {},
-		"namespace": {},
-		"pod":       {},
-		"container": {},
-		"comm":      {},
-		"tgid":      {},
-		"pid":       {},
+	availableColumns := map[string]int{
```

Comment:
Maybe add a comment indicating why to use 0 as width?

Reply:
I created two wrapper functions to simplify the parser API and keep this logic of width/taps inside the parser, and it is transparent for the gadgets:

https://github.com/kinvolk/inspektor-gadget/blob/f1870b7f3c3a6f61cba3d48a00286632d16910d9/cmd/common/utils/parser.go#L46-L65

WDYT?

Reply:
I like this idea of having to different functions, hence we don't need to pass "true/false" from all the callers.